docs/source/fitting/fitfunctions/Redfield.rst

```diff
@@ -9,30 +9,41 @@ Redfield
 Description
 -----------
 
-The Redfield formula for the LF dependence on the muon spin relaxation rate.
+The Redfield formula for the Longitudinal Field (LF) dependence of the muon spin relaxation rate, :math:`\Lambda`, given in units of
+:math:`\mu s^{-1}`, with the applied longitudinal magnetic field, for given local magnetic field (:math:`H_\text{loc}`) and correlation time of
+fluctuations at muon spin sites (:math:`\tau`) is:
 
 .. math:: \Lambda(t)= \frac{2\gamma^2_\mu H^2_\text{loc}\tau}{1+\gamma^2_\mu H^2_\text{LF} \tau^2}
 
 where,
 
-:math:`H_\text{loc}` is the local magnetic field,
+:math:`H_\text{loc}` is the local magnetic field, in Gauss,
 
-:math:`H_\text{LF}` is the longitudinal magnetic field applied,
+:math:`H_\text{LF}` is the applied longitudinal magnetic field, in Gauss,
 
-and its expression is given as :math:`\tau = \frac{1}{f}`
+:math:`\tau' is the muon spin correlation time, in microseconds, with expression given as :math:`\tau = \frac{1}{f}`
 where :math:`f` is the frequency of fluctuation at muon sites.
 
+And :math:`\gamma_\mu` is the gyromagnetic ratio of the muon spin, given in units of :math:`[rad]x\frac{MHz}{Gauss}`
 .. plot::
```

Comment:
I think we also need a space between these two lines 
![image](https://github.com/mantidproject/mantid/assets/40830825/db6d7097-8772-436b-b373-e2ab32eef989)
```suggestion
And :math:`\gamma_\mu` is the gyromagnetic ratio of the muon spin, given in units of :math:`[rad]x\frac{MHz}{Gauss}`

.. plot::
```